### PR TITLE
[Federated] Split Ansible playbooks; Use network names

### PR DESCRIPTION
**Pull request by KPrasch**
- Separates server provisioning from Ursula network services configuration
- Uses `NUCYPHER_NETWORK_NAME` envvar as dynamic inventory tag
- Updates .gitignore with Ansible variables

### CODE REVIEW

File: deploy/ansible/provision.yml [hunk 1]
```diff
@@ -14,159 +16,30 @@
...
         count_tag:
-          Role: "{{ ec2_tag_Role }}"
+          Role: "{{ nucypher_network_name }}"
         volumes:
           - device_name: /dev/xvda
             volume_type: gp2
             volume_size: "{{ ec2_volume_size }}"
             delete_on_termination: true
-      register: ec2
+      register: nucypher_swarm
```
Comment (EvaLovlac3) [inline deploy/ansible/provision.yml]: Why this name?

File: deploy/ansible/run_seednodes.yml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    - name: "Install Python Dependencies via Pipenv"
+      shell: "pipenv install --three --dev --pre --skip-lock"
+      args:
+        chdir: ./code
+      environment:
+        LC_ALL: C.UTF-8
+        LANG: C.UTF-8
+
+    - name: "Generate Ursula passphrase"
+      shell: head -c 32 /dev/urandom | sha256sum
```
[the remark this answers is not in the document]
Comment (jMyles) [reply deploy/ansible/run_seednodes.yml]: Just to make it more human readable / pronouncable?

FWIW, we actually need to be using `getrandom` here (at least, in my view).  Does pwgen use `getrandom`?

I think it's actually better to do this step in python.

**Comment by jMyles** [comment]
This looks all right to me.  But what's the actual plan here?  This is going into federated... and then what?

File: deploy/ansible/run_seednodes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (derekpierre) [reply deploy/ansible/run_seednodes.yml]: Side note: @KPrasch, I'm not sure if you care or not, but the `sha256sum` command typically prints the filename after the hash. In the case of `stdin`, it prints `-`, so the password currently contains a space and the dash.
```
$ head -c 32 /dev/urandom | sha256sum
2a80abdd420da6a0ea49a662396ff2f657dd467c19de09b6854e6552a35121d7  -
```

Doesn't matter really, but if you do care, you can remove it using `awk`
```
$ head -c 32 /dev/urandom | sha256sum | awk '{print $1}'
e85545a4227c5c089b93c7ef9edbf0dfb9c853177dbfd903d74c01f926531086
```

File: deploy/ansible/provision.yml (hunk 1 above)
Comment (cygnusv) [reply deploy/ansible/provision.yml]: Yeah, I thought "fleets" was our thing.

File: deploy/ansible/run_seednodes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (KPrasch) [reply deploy/ansible/run_seednodes.yml]: Heh - Good catch man, those extras characters are pretty annoying!  I think we can do both of the above steps:

- Use @derekpierre suggestion with awk
- Later on move the step to python (there is a growing stub for this ion the codebase)

**Comment by KPrasch** [comment]
Yep - Good question - currently this set of playbooks is incompatible with `master` - Which I suggest we handle in a later PR.

File: deploy/ansible/provision.yml (hunk 1 above)
Comment (KPrasch) [reply deploy/ansible/provision.yml]: oh yeah, a bit of unintentional name-shadowing here

File: deploy/ansible/run_seednodes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (KPrasch) [reply deploy/ansible/run_seednodes.yml]: :heavy_check_mark:

File: deploy/ansible/provision.yml (hunk 1 above)
Comment (KPrasch) [reply deploy/ansible/provision.yml]: :heavy_check_mark:

**Pull request merged**